pkg/operators/oci-handler/oci.go

```diff
@@ -266,11 +266,20 @@ func (o *OciHandlerInstance) init(gadgetCtx operators.GadgetContext) error {
 }
 
 func (o *OciHandlerInstance) Start(gadgetCtx operators.GadgetContext) error {
+	started := []operators.DataOperatorInstance{}
```

Comment:
```suggestion
	started := []operators.ImageOperatorInstance{}
```

Reply:
Good catch! The compiler didn't say anything!

Reply:
It's because the DataOperatorInstance implements a subset of ImageOperatorInstance's methods. The other way around wouldn't have worked 😅